packages/playwright-test/src/reporters/junit.ts

```diff
@@ -133,6 +133,94 @@ class JUnitReporter implements Reporter {
     };
     entries.push(entry);
 
+    // Xray Test Management supports testcase level properties, where additional metadata may be provided
+    // some annotations are encoded as value attributes, other as cdata content
+    const properties: XMLEntry = {
+      name: 'properties',
+      children: [] as XMLEntry[]
+    };
+
+    const textValueBasedAnnotations = ['test_id', 'test_key', 'test_summary', 'requirements'];
```

Comment:
I am hesitant supporting only some annotations. Why don't we put all annotations as properties? Should be safe enough, as long as Xray ignores unknown properties, which it probably does?

Reply:
Well, this annotations are the ones Xray understands as text based values. We could make it more generic (please see next comment)